cmd/tracee-rules/main.go

```diff
@@ -140,10 +142,52 @@ func main() {
 
 			}
 
+			if configPath != "" {
+				watcher, err := fsnotify.NewWatcher()
+				if err != nil {
+					return err
+				}
+				defer watcher.Close()
+				go func() {
+					for {
+						select {
+						case event, ok := <-watcher.Events:
+							if !ok {
+								return
+							}
+							if event.Op&fsnotify.Write == fsnotify.Write {
+								fmt.Printf("\n*** Configuration Changed ***\nFile: %v\n\n", event)
+								_, err := celsig.LoadFromConfigFile(configPath)
```

Comment:
Why don't we update the signatures?

Reply:
I tried but Engine.UnloadSignature followed by Engine.LoadSitnagure has some indeterministic behaviour. Taking it out of scope of this PR, but eventually we may want to reload CEL signatures whenever config file has changes.